### ISSUE TITLE
Not all multi-line responses are dealt with properly

### ISSUE THREAD

**Issue by AlphabetsAlphabets** [AlphabetsAlphabets/rust-ftp-client#2]
An example would be the "HELP" command. It doesn't have the code followed by hypen mark. It just outputs the values like this:

```
214-The following commands are recognized.
 ABOR ACCT ALLO APPE CDUP CWD  DELE EPRT EPSV FEAT HELP LIST MDTM MKD
 MODE NLST NOOP OPTS PASS PASV PORT PWD  QUIT REIN REST RETR RMD  RNFR
```

Because of the way code is currently being handled. Another `receive_response` will need to be called for the full message to be shown. As far as I'm aware, only `HELP` has this so far.

**Comment by AlphabetsAlphabets** [comment]
Turns out that in order to get the full response of the "HELP" command you need five calls to `receive_response`.

```rust
    send_msg(&mut stream, "HELP");
    let response = receive_response(&mut stream);
    println!("{}", response);
    let response = receive_response(&mut stream);
    println!("{}", response);
    let response = receive_response(&mut stream);
    println!("{}", response);
    let response = receive_response(&mut stream);
    println!("{}", response);
    let response = receive_response(&mut stream);
    println!("{}", response);
```

Currently, this the output for a multi-line *command*.

```
214-The following commands are recognized.
 ABOR ACCT ALLO APPE CDUP CWD  DELE EPRT EPSV FEAT HELP LIST MDTM MKD

 MODE NLST NOOP OPTS PASS PASV PORT PWD  QUIT REIN REST RETR RMD  RNFR

 RNTO SITE SIZE SMNT STAT STOR STOU STRU SYST TYPE USER XCUP XCWD XMKD

 XPWD XRMD

214 Help OK.
```

The trend is that instead of `123-` it would be `' '<output>`, it's gonna be a good idea to take a look at what's at the end of each line. If it's just `\r\n` or `\n`. Should only look for the above if `' '<output>` pattern is found.

The calls to `receive_response` need to be present as well. Otherwise, the content of the previous response is what's left.

**Comment by AlphabetsAlphabets** [comment]
Fixed in 47240fb9c5635201b93298ee1c966bc76f9a96c5
